tests/unit/test_unit.py

```diff
@@ -2590,7 +2140,7 @@ def test_11():
                     "nucleotide_index": 3,
                     "gene_name": "C",
                     "gene_position": 4,
-                    "codon_idx": 2,
+                    "codon_idx": 0,
```

Comment:
why are the codon indexes changing around the place?

Reply:
It's actually a gumpy bug which this fixes. In some cases gumpy seems to give off-by-1 codon indices

e.g in this example it is definitely 0 rather than 2:
```
gene_pos   1  2  3  4
codon_idx  0  1  2  0
```